### PR TITLE
coleco.xml: Added 27 working items + 7 not working

### PR DESCRIPTION
**Pull request by ArcadeShadow**
New working software list additions
--------------------------------------------
Anti-Environment Encounter [ColecoVision Addict]
Arno Dash [ColecoVision Addict]
Bank Panic [ColecoVision Addict]
Burn Rubber [ColecoVision Addict]
C-So! [ColecoVision Addict]
Caterpillar S.O.S. [ColecoVision Addict]
Champion Pro Wrestling [ColecoVision Addict]
Diamond Dash II [ColecoVision Addict]
Flapee Byrd [ColecoVision Addict]
Klondike Solitaire (v1.0) [ColecoVision Addict]
Kobashi [ColecoVision Addict]
Mahjong Solitaire (v1.0) [ColecoVision Addict]
Mario Bros. [ColecoVision Addict]
Minesweeper (v1.0) [ColecoVision Addict]
Mr Chin [ColecoVision Addict]
Pegged (v1.0) [ColecoVision Addict]
Pillars [ColecoVision Addict]
Pitman [ColecoVision Addict]
Pooyan [ColecoVision Addict]
Pyramid Warp + Battleship Clapton II [ColecoVision Addict]
Snake [ColecoVision Addict]
Spunky's Super Car! [ColecoVision Addict]
Vexed [ColecoVision Addict]
Winky Trap [ColecoVision Addict]
Zippy Race [ColecoVision Addict]
Zombie Calavera Prologue [ColecoVision Addict]
Zombie Near [ColecoVision Addict]

New NOT working software list addition
--------------------------------------------
Booming Boy [ColecoVision Addict]
Boxxle [ColecoVision Addict]
Buck Rogers Super Game [ColecoVision Addict]
Caos Begins [ColecoVision Addict]
Caverns of Titan [ColecoVision Addict]
Kung-Fu Master [ColecoVision Addict]
The Black Onyx [ColecoVision Addict]

### CODE REVIEW

File: hash/coleco.xml [hunk 1]
```diff
@@ -3420,4 +3420,386 @@ license:CC0-1.0
 		</part>
 	</software>
 
+<!-- Aftermarket releases -->
```
Comment (cuavas) [inline hash/coleco.xml:3423]: What do you mean by “Aftermarket”?  The term refers to any parts sold to be fitted to a machine/system after it’s purchased.  The archetypal example is an “aftermarket sunroof”, i.e. an opening roof fitted to a vehicle after delivery (as opposed to a “factory sunroof”, fitted to the car before delivery).  Any cartridges or peripherals that weren’t included as pack-ins with the system are “aftermarket”.

Comment (ArcadeShadow) [reply hash/coleco.xml:3423]: I intend to indicate the games made available after the commercial life of the ColecoVision system. It was for this reason that I do not mix with the others. Perhaps there is a better term for this.

**Comment by Gurudumps** [comment]
Arcade Shadow: The correct term is 'homebrew'. Or perhaps 'rip-offs' would be a better describe it as several have been sold in the past where the person who did it definitely does not own the IP or assets for the games being copied.

**Comment by ArcadeShadow** [comment]
Changed comment

**Comment by happppp** [comment]
Sorry to make you change it again: Rip-off is not a good swlist category. Homebrew is good.

**Comment by Gurudumps** [comment]
Hehe! He didn't get the humor in that comment. Although it's not untrue.... there is a surprisingly large number of homebrew titles being sold that are straight copies/ports from other commercial games, especially from the SG1000.
But yeah homebrew is the right word.

**Comment by happppp** [comment]
Yeah, converting a game from another console (changing a few I/O addresses and whatnot) and then selling it instead of releasing a free romhack patch is bootlegging.

BTW is it normal for them to provide non-2^x size files?

**Comment by Gurudumps** [comment]
Well.... it's *normal* for amateurs to not know why things need to be power-of-2 ;-)
They just converted/hacked the games and didn't pad them out to the proper ROM size. Some of them are power-of-2 but most are not padded.

**Comment by ArcadeShadow** [comment]
Changed to "Homebrew releases"

**Pull request merged**